sktime/forecasting/base/_base.py

```diff
@@ -105,6 +105,7 @@ def fit(self, y, X=None, fh=None):
         self._X = X
         self._y = y
 
+        self._set_cutoff(y.index[-1])
```

Comment:
this is now included in `_base`, make sure you update from `main`

Reply:
I will do that